packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -51,7 +51,25 @@ export class CdkErrorMapper {
       classification: 'ERROR',
     },
     {
-      errorRegex: /\[ERR_MODULE_NOT_FOUND\]:(.*)\n/,
+      errorRegex: /Unable to resolve AWS account to use/,
+      humanReadableErrorMessage:
+        'Unable to resolve AWS account to use. It must be either configured when you define your CDK Stack, or through the environment',
+      resolutionMessage:
+        'You can retry your last request as this is most likely a transient issue: https://github.com/aws/aws-cdk/issues/24744',
```

Comment:
I've definitely gotten this error when it was not transient (eg if I have stale local creds). I'd include something here about checking that local credentials are valid / refreshed.

Reply:
I'll add it, but were you getting this exact `Unable to resolve AWS account to use` or `Unable to resolve AWS credentials`? From the mentioned github issue, it seems a widespread issue.

Reply:
Ah maybe I was getting "Unable to resolve AWS credentials". Regardless, the new message looks good.